go.mod

```diff
@@ -26,6 +26,7 @@ require (
 	google.golang.org/protobuf v1.30.0
 )
 
+replace github.com/cloudquery/plugin-pb-go v1.0.8 => ../plugin-pb-go
```

Comment:
We will need to update this once is https://github.com/cloudquery/plugin-pb-go/pull/5 is merged and released